addons/udes_sale_stock/models/sale_order.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    # Rename states
+    state = fields.Selection([
+        ('draft', 'Quotation'),
+        ('sent', 'Quotation Sent'),
+        ('sale', 'In Progress'),
+        ('done', 'Done'),
+        ('cancel', 'Cancelled'),
+    ])
```

Comment:
Could this use selection_add instead of redefining the existing selection values?

Reply:
The idea is to keep the same values, but just renaming their names, I'll check if we can use selection_add to do it.

Reply:
Yes, looks like selection_add will replace the ones we change the name. I will update it.
Thanks.